staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/kms_plugin_mock.go

```diff
@@ -16,39 +16,38 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package master
+package envelope
```

Comment:
suggest separating this into a testing or mock or fake package to make it really clear this isn't a real implementation

Reply:
Done.